examples/platformvm/README.md

```diff
@@ -12,4 +12,5 @@ Tests for the Avalanche [PlatformVM RPC](https://docs.avax.network/build/avalanc
 * [createKeypair.ts](./createKeypair.ts)
 * [createSubnetTx.ts](./createSubnetTx.ts)
 * [exportTx-xchain.ts](./exportTx-xchain.ts)
+* [getRewardUTXOs.ts](./getRewardUTXOs.ts)
```

Comment:
I think a pointer to `getStake` might be missing here

Reply:
Nice catch. Resolved.